replace.go

```diff
@@ -64,6 +71,10 @@ func (r *Runner) ReplaceWithRendered(name, chartName, chartPath string, o Replac
 	if o.Namespace != "" {
 		additionalFlags += createFlagChain("namespace", []string{o.Namespace})
 	}
+	additionalFlags += createFlagChain("api-versions", o.ApiVersions)
+	if len(o.ApiVersions) > 0 {
+		additionalFlags += " " + strings.Join(o.ApiVersions, " ")
+	}
```

Comment:
Thanks! Just confirming but how did you test this?

I thought `--api-versions` was a StringArray flag, which means that it accepts multiple values by repeating the flag, like `--api-versions aaa --api-versions bbb`.
If that's true L75-L7 would be redundant, and this block of code should be modified to look more like:

```suggestion
	additionalFlags += createFlagChain("api-versions", o.ApiVersions)
```

Reply:
Ah sorry my bad I misread the code above where setValues and SetFlags are built that way but they are entirely different options and not the same. You are entirely right, the option can be used multiple times and a single createFlagChain should be enough to consume that.

I'll amend the code in a minute.

Regarding the matter of testing it I am at a loss currently - there might be a way to build a new helmfile version with my changes but I am yet unaware how to do that.

Reply:
@Arabus Thanks for confirming!

Re: testing, you can swap helmfile's dependency to vals by adding the following replace statement to `go.mod` of helmfile, assuming you've git-clone `helmfile` and `vals` as directories within the same parent directory:

`./helmfile/go.mod`:
```
replace github.com/variantdev/vals => ../vals
```

and run `cd ./helmfile; go build .` so that you get a executable binary at `./helmfile` that can be run like `./helmfile apply`.